### PR TITLE
add install notes on rnpm, fix type on response type desc

### PR DESCRIPTION
**Pull request by bsy**
@chirag04

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -14,11 +14,11 @@ A react-native wrapper for handling in-app purchases.
...
 1. Run `npm install react-native-in-app-utils --save`.
 
-2. Open your project in XCode, right click on `Libraries`, click `Add Files to "Your Project Name"` and add `InAppUtils.xcodeproj`. (situated in `node_modules/react-native-in-app-utils`) [(This](http://url.brentvatne.ca/jQp8) then [this](http://url.brentvatne.ca/1gqUD), just with InAppUtils).
+2. Make sure you have `rnpm` installed: `npm install rnpm -g`
 
-3. Link `libInAppUtils.a` with your Libararies. To do that, click on your project folder, select `Build Phases` in the top bar, scroll to `Link Binary with Libraries`, press the `+` at the very bottom and add `libInAppUtils.a` from the `node_modules/react-native-in-app-utils/InAppUtils` folder. [(Screenshot)](http://url.brentvatne.ca/17Xfe).
+3. Run `rnpm link react-native-in-app-utils`
 
-4. Whenever you want to use it within React code now you just have to do: `var InAppUtils = require('NativeModules').InAppUtils;`
+4. Whenever you want to use it within React code now you just have to do: `var InAppUtils = require('NativeModules').InAppUtils;` or for ES6: `import { InAppUtils } from 'NativeModules'`
```
Comment (chirag04) [inline Readme.md]: import from `React.NativeModules`?

**Comment by chirag04** [comment]
just one nit otherwise looks great. Thanks for the adding rnpm.

**Comment by bsy** [comment]
Why would we need that if it wasn't required in the original require
syntax?  Works fine with out adding `React.` in front of NativeModules for
me.

--ben

**Comment by chirag04** [comment]
That readme was written long time ago and it's generally not a good idea to assume `nativeModules` available in global namespace.

Refer: https://facebook.github.io/react-native/docs/native-modules-ios.html#content

**Pull request closed without merging**